### DISCUSSION TITLE
How to upgrade to the latest version?

### DISCUSSION THREAD

**Discussion by temlekur** [mitchmac/ServerlessWP#41] [Q&A]
Has anyone upgraded to the latest version 6.4.2 yet? 
Please share your experience how to do it?

**Comment by mitchmac** [comment]
Hi, I haven't tried it yet but you should be able to download the latest WordPress release and put the files in /wp then commit the files to your repository.

You would want to make sure wp-config.php is correct after doing so and would want to make sure the mu-plugins directory includes serverlesswp.php.

I will try to make the WordPress version update here in this repository in a few days.
